src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java

```diff
@@ -402,7 +404,16 @@ private static boolean isSingleMap(Map<String, ?> args) {
         }
 
         public String getSymbol() {
-            return real instanceof StepDescriptor ? ((StepDescriptor) real).getFunctionName() : real.getClass().getAnnotation(Symbol.class).value()[0];
+            if (real instanceof StepDescriptor) {
+                return ((StepDescriptor) real).getFunctionName();
+            } else {
+                Set<String> symbolValues = SymbolLookup.getSymbolValue(real);
+                if (!symbolValues.isEmpty()) {
+                    return symbolValues.iterator().next();
+                } else {
```

Comment:
Should not happen, since we have already checked for this above; could just as well throw an `AssertionError`.